### PR TITLE
contribution overview process added

### PR DESCRIPTION
**Pull request by pinarpink**
Adresses https://github.com/elixir-europe/rdm-toolkit/issues/19

Some work still needs to be done though.

### CODE REVIEW

**Comment by bedroesb** [comment]
is it possible to export the diagram to svg instead of png? :)

**Comment by pinarpink** [comment]
@bedroesb  ok let me do svg

**Comment by pinarpink** [comment]
@bedroesb   this should be good to go. let me know.

File: pages/contribute/how_to_contribute.md [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
 
-## Subtitle 1
+## Ways of contributing 
+
+We accept contributions in two ways sketched below, through GitHub or through Google Docs. <br/>
+ 
+![image](images/howto_contribute_overview.svg) <br/>
+
+<br/>
+* **Github way:** We use git to develop the RDM toolkit website. Git allows us to accept and coordinate contributions under an open authoring and review process. Therefore, GitHub is our preferred way of accepting contributions. To begin with, you'll need a personal GitHub account, which you can create [here](https://github.com/join).
```
Comment (nsoranzo) [inline pages/contribute/how_to_contribute.md:18]:
```suggestion
* **Github way:** We use GitHub to develop the RDM toolkit website. GitHub allows us to accept and coordinate contributions under an open authoring and review process. Therefore, this is our preferred way of accepting contributions. To begin with, you'll need a personal GitHub account, which you can create [here](https://github.com/join).   
```

File: pages/contribute/how_to_contribute.md [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
-## Subtitle 1
+## Ways of contributing 
+
+We accept contributions in two ways sketched below, through GitHub or through Google Docs. <br/>
+ 
+![image](images/howto_contribute_overview.svg) <br/>
+
+<br/>
+* **Github way:** We use git to develop the RDM toolkit website. Git allows us to accept and coordinate contributions under an open authoring and review process. Therefore, GitHub is our preferred way of accepting contributions. To begin with, you'll need a personal GitHub account, which you can create [here](https://github.com/join).   
+* **Google Doc way:** For those contributors that are not versed in git, we provide Google Doc templates. This approach requires e-mail communication with our editors, who will be responsible for transferring the content to our git repository on your behalf.
```
Comment (nsoranzo) [inline pages/contribute/how_to_contribute.md:19]:
```suggestion
* **Google Doc way:** Alternatively, we provide Google Doc templates. This approach requires e-mail communication with our editors, who will be responsible for transferring the content to our git repository on your behalf.
```

**Comment by bedroesb** [comment]
Looks good! Only comment I have is that in order to let the image show up correctly you have to change `image <br/>` to `{% include image.html file="howto_contribute_overview.svg" caption="Figure 1. How to contribute overview." alt="How to contribute overview" %}`

+ I don't think the </br> are needed in this case when you leave a whitespace in between :) 

For more information about the syntax you can always visit the [demo page](https://elixir-europe.github.io/rdm-toolkit/demo_page.html)

File: pages/contribute/how_to_contribute.md [hunk 3]
```diff
@@ -1,12 +1,27 @@
...
+summary: Here you'll find an overview of the ways of contributing to this website. Depending on your chosen way, please follow the links on the left, where you'll find step-by-step instructions.  
 ---
 
 
-## Subtitle 1
+## Ways of contributing 
+
+We accept contributions in two ways sketched below, through GitHub or through Google Docs. <br/>
+ 
+![image](images/howto_contribute_overview.svg) <br/>
```
Comment (bedroesb) [inline pages/contribute/how_to_contribute.md:15]:
```suggestion
{% include image.html file="howto_contribute_overview.svg" alt="How to contribute overview" %}
```

File: pages/contribute/how_to_contribute.md (hunk 1 above)
Comment (bedroesb) [inline pages/contribute/how_to_contribute.md:18]:
```suggestion
* **[Github way:](github_way)** We use git to develop the RDM toolkit website. Git allows us to accept and coordinate contributions under an open authoring and review process. Therefore, GitHub is our preferred way of accepting contributions. To begin with, you'll need a personal GitHub account, which you can create [here](https://github.com/join).   
```

File: pages/contribute/how_to_contribute.md (hunk 2 above)
Comment (bedroesb) [inline pages/contribute/how_to_contribute.md:19]:
```suggestion
* **[Google Doc way:](google_doc_way)** For those contributors that are not versed in git, we provide Google Doc templates. This approach requires e-mail communication with our editors, who will be responsible for transferring the content to our git repository on your behalf.
```
I added the links to the separate pages

File: pages/contribute/how_to_contribute.md [hunk 4]
```diff
@@ -1,12 +1,27 @@
...
+
+<br/>
```
Comment (bedroesb) [inline pages/contribute/how_to_contribute.md:17]:
```suggestion
```

File: pages/contribute/how_to_contribute.md [hunk 5]
```diff
@@ -1,12 +1,27 @@
...
-summary: This is a test summary
+toc: false
+summary: Here you'll find an overview of the ways of contributing to this website. Depending on your chosen way, please follow the links on the left, where you'll find step-by-step instructions.  
 ---
 
 
-## Subtitle 1
+## Ways of contributing 
+
+We accept contributions in two ways sketched below, through GitHub or through Google Docs. <br/>
```
Comment (bedroesb) [inline pages/contribute/how_to_contribute.md:13]:
```suggestion
We accept contributions in two ways sketched below, through GitHub or through Google Docs.
```

**Review by bedroesb — CHANGES_REQUESTED** [review]

**Review by bedroesb — APPROVED** [review]

**Pull request merged**